packages/analytics/analytics-metric-provider/src/components/MetricsProvider.vue

```diff
@@ -59,24 +59,21 @@ if (props.dimension && queryableExploreDimensions.findIndex(x => x === props.dim
 const queryBridge: AnalyticsBridge | undefined = inject(INJECT_QUERY_PROVIDER)
 
 let queryFn: AnalyticsBridge['queryFn']
-let evaluateFeatureFlagFn: AnalyticsBridge['evaluateFeatureFlagFn']
 
 if (!queryBridge) {
   console.warn('Analytics dashboards require a query bridge supplied via provide / inject.')
   console.warn("Please ensure your application has a query bridge provided under the key 'analytics-query-provider', as described in")
   console.warn('https://github.com/Kong/public-ui-components/blob/main/packages/analytics/analytics-metric-provider/README.md#requirements')
   queryFn = () => Promise.reject(new Error('Query bridge required'))
-  evaluateFeatureFlagFn = (_key, defaultValue) => defaultValue
 } else {
   queryFn = queryBridge.queryFn
-  evaluateFeatureFlagFn = queryBridge.evaluateFeatureFlagFn
 }
 
 const analyticsConfigStore = useAnalyticsConfigStore()
 
 // Check if the current org has long enough retention to make a sane trend query.
 // If the feature flag is set, trend access is always true.
-const hasTrendAccess = computed<boolean>(() => skuFeatureFlag.value || analyticsConfigStore.longRetention)
```

Comment:
This should now always return true.  I'm thinking that it probably isn't straightforward to carry this whole thing throughout the metric cards, and we probably don't strictly need to, so it's probably fine to just return computed true.